fastlane_bot/utils.py

```diff
@@ -282,15 +282,3 @@ def find_latest_timestamped_folder(logging_path=None):
 
     list_of_folders.sort(reverse=True)  # Sort the folders in descending order
     return list_of_folders[0]  # The first one is the latest
-
-
-def count_bytes(data: HexBytes) -> (int, int):
```

Comment:
just pointing out that this is not the docstring format we want to use in the main library